albumentations/augmentations/functional.py

```diff
@@ -620,6 +620,16 @@ def shift_rgb(img, r_shift, g_shift, b_shift):
     return _shift_rgb_non_uint8(img, r_shift, g_shift, b_shift)
 
 
+@clipped
+def linear_transformation_rgb(img, transformation_matrix, shifts=None):
+    result_img = cv2.transform(img, transformation_matrix.astype(np.float32))
```

Comment:
You use there shifts, but in ToSepia class you do not use it.
I think, it is not needed, because we have RGBShift transform and this pipeline can do similar thinks:
```
A.Compose([
    A.Sepia(),
    A.RGBShift()
])
```

Reply:
You are right. `shifts`  is not needed for sepia and can be deleted for now, but I have added it because I wanted to have full linear transformation function in channel space (y = Ax + b, instead of separate y = Ax and y = x + b) that can be used somewhere else.

Your approach using Compose - is ok, I have thought about it, but it is not 100% precise because of `@clipped` - if I understand it correctly it will clip everything before RGB shift. So there can be some mistakes.